drop-transfer/src/transfer.rs

```diff
@@ -72,6 +72,10 @@ impl<F: File> TransferData<F> {
         Self::new_with_uuid(peer, files, Uuid::new_v4(), config)
     }
 
+    pub(crate) fn peer(&self) -> IpAddr {
+        self.peer
+    }
+
```

Comment:
@LukasPukenis there is already this method in the `Transfer` trait which is implemented on `TransferData<F>`
```suggestion

```